app/main.py

```diff
@@ -16,8 +16,8 @@ def index():
 	return render_template('index.html')
 
 def generate_badges():
-    os.system(os.path.join(app.config['UPLOAD_FOLDER'], 'merge_badges.sh'))
-
+    os.system(SCRIPT)
+    
```

Comment:
Why is this change needed?

Reply:
codacy was showing error in that previously